### ISSUE TITLE
Array of device tokens not worked

### ISSUE THREAD

**Issue by asvechkar** [semyonchetvertnyh/laravel-apn-notification-channel#3]
In User model add method:
`
/**
     * Route notifications for the APN channel.
     *
     * @return string|array
     */
    public function routeNotificationForApn()
    {
        return $this->mobile_devices()->platform('ios')->get()->pluck('device_token');
    }
`
Bun return Exception:
`
Symfony\Component\Debug\Exception\FatalThrowableError  : Argument 2 passed to Pushok\Notification::__construct() must be of the type string, array given, called in /Users/asvechkar/Projects/.../vendor/semyonchetvertnyh/laravel-apn-notification-channel/src/ApnChannel.php on line 77

  at /Users/asvechkar/Projects/.../vendor/edamov/pushok/src/Notification.php:71
    67|      *
    68|      * @param Payload $payload
    69|      * @param string $deviceToken
    70|      */
  > 71|     public function __construct(Payload $payload, string $deviceToken)
    72|     {
    73|         $this->payload = $payload;
    74|         $this->deviceToken = $deviceToken;
    75|     }

  Exception trace:

  1   Pushok\Notification::__construct(Object(Pushok\Payload))
      /Users/asvechkar/Projects/.../vendor/semyonchetvertnyh/laravel-apn-notification-channel/src/ApnChannel.php:77

  2   SemyonChetvertnyh\ApnNotificationChannel\ApnChannel::SemyonChetvertnyh\ApnNotificationChannel\{closure}("\*\items")
      [internal]:0

`

**Comment by semyonchetvertnyh** [comment]
Are you sure you return a correct array of device tokens in ``routeNotificationForApn()`` method?

It seems like you return something like:
```php
public function routeNotificationForApn()
 {
    return [
        ['2c631bc92db7d2ad7ddf05ac7123e2fcfee6abea8de2d0fa642a1ab6727d9643'],
        ['43c44b308cf02dabfa7fdfecf1828f7859c339aa0ee56688a2599fdae979fc23'],
        ['8feee8f9a7700f34ba1f100421ef81709736c7530f65be4a11ea02478503f090'],
    ];
}
```

But it must be:
```php
public function routeNotificationForApn()
 {
    return [
        '2c631bc92db7d2ad7ddf05ac7123e2fcfee6abea8de2d0fa642a1ab6727d9643',
        '43c44b308cf02dabfa7fdfecf1828f7859c339aa0ee56688a2599fdae979fc23',
        '8feee8f9a7700f34ba1f100421ef81709736c7530f65be4a11ea02478503f090',
    ];
}
```

**Comment by semyonchetvertnyh** [comment]
In your case you should add ```->all()``` to the end to return an array, not a collection:

```php
public function routeNotificationForApn() {
    return $this->mobile_devices()->platform('ios')->get()->pluck('device_token')->all();
}
```

**Comment by semyonchetvertnyh** [comment]
Anyway, thanks for issue! I fixed this behaviour in v0.1.4.

**Comment by asvechkar** [comment]
Thanx.
